src/TSTransformer/nodes/jsx/transformJsx.ts

```diff
@@ -19,15 +16,7 @@ export function transformJsx(
 	attributes: ts.JsxAttributes,
 	children: ReadonlyArray<ts.JsxChild>,
 ) {
-	if (state.compilerOptions.jsxFactory !== "Roact.createElement") {
-		DiagnosticService.addSingleDiagnostic(errors.invalidJsxFactory(node));
-		return luau.none();
-	}
-
-	const isFragment =
-		state.services.roactSymbolManager &&
-		state.typeChecker.getSymbolAtLocation(tagName) ===
-			state.services.roactSymbolManager.getSymbolOrThrow(ROACT_SYMBOL_NAMES.Fragment);
+	const isFragment = false; // TODO?
```

Comment:
Is this still necessary as compiler code? It seems more suited to do this in the Roact package (or remove the intrinsic fragment option entirely and just recommend `<></>`)